hostapd/Makefile

```diff
@@ -24,9 +24,14 @@ endif
 CFLAGS += $(EXTRA_CFLAGS)
 CFLAGS += -I$(abspath ../src)
 CFLAGS += -I$(abspath ../src/utils)
+CFLAGS += -DCONFIG_INF_AUTH
 
 export BINDIR ?= /usr/local/bin/
 
+ifdef CONFIG_INF_AUTH
+CLFAGS += -DCONFIG_INF_AUTH
+endif
+
```

Comment:
There is a bug in the conditional definition of CONFIG_INF_AUTH flag, the conditional definition is not working. Temporarily hard coding the definition of "CONFIG_INF_AUTH" flag in line 27 till I find out the issue.